demos/classification_benchmark_demo/cpp/main.cpp

```diff
@@ -26,90 +26,93 @@
+DEFINE_uint32(nt, 5, nt_msg);
+
+constexpr char nthreads_msg[] = "specify count of threads";
+DEFINE_uint32(nthreads, 0, nthreads_msg);
+
+constexpr char res_msg[] = "set image grid resolution in format WxH. Default value is 1280x720";
+DEFINE_string(res, "1280x720", res_msg);
+
+constexpr char show_msg[] = "disable showing of processed images";
+DEFINE_bool(show, true, show_msg);
+
+constexpr char time_msg[] = "time in seconds to execute program. Default is -1 (infinite time)";
+DEFINE_uint32(time, std::numeric_limits<gflags::uint32>::max(), time_msg);
+
+constexpr char u_msg[] = "list of monitors to show initially";
+DEFINE_string(u, "", u_msg);
+
+void parse(int argc, char *argv[]) {
+    gflags::ParseCommandLineFlags(&argc, &argv, false);
+    if (FLAGS_h || 1 == argc) {
+        std::cout <<   "\t[ -h]                 " << h_msg
+                  << "\n\t  -i <INPUT>          " << i_msg
+                  << "\n\t --labels <LABELS>    " << labels_msg
+                  << "\n\t  -m <MODEL FILE>     " << m_msg
+                  << "\n\t[--auto_resize]       " << res_msg
+                  << "\n\t[ -d <DEVICE>]        " << d_msg
+                  << "\n\t[--gt <STRING>]       " << gt_msg
+                  << "\n\t[--layout <STRING>]   " << layout_msg
+                  << "\n\t[--nireq <NUMBER>]    " << nireq_msg
+                  << "\n\t[--nstreams <NUMBER>] " << nstreams_msg
+                  << "\n\t[--nt <NUMBER>]       " << nt_msg
+                  << "\n\t[--nthreads <NUMBER>] " << nthreads_msg
+                  << "\n\t[--res <STRING>]      " << res_msg
+                  << "\n\t[--show] ([--noshow]) " << show_msg
+                  << "\n\t[--time <NUMBER>]     " << time_msg
+                  << "\n\t[ -u <DEVICE>]                 " << u_msg
+                  << "\n\tKey bindings:"
+                     "\n\t\tQ, q, Esc - Quit"
+                     "\n\t\tR, r, SpaceBar - Restart testing"
+                     "\n\t\tC - average CPU load, D - load distribution over cores, M - memory usage, H - hide\n";
 
-bool ParseAndCheckCommandLine(int argc, char *argv[]) {
-    // ---------------------------Parsing and validation of input args--------------------------------------
-    gflags::ParseCommandLineNonHelpFlags(&argc, &argv, true);
-    if (FLAGS_h) {
-        showUsage();
         showAvailableDevices();
-        return false;
-    }
-
-    if (FLAGS_i.empty()) {
-        throw std::logic_error("Parameter -i is not set");
-    }
-
-    if (FLAGS_m.empty()) {
-        throw std::logic_error("Parameter -m is not set");
-    }
-
-    if (FLAGS_labels.empty()) {
-        throw std::logic_error("Parameter -labels is not set");
+        slog::info << ov::get_openvino_version() << slog::endl;
+        exit(0);
+    } if (FLAGS_i.empty()) {
+        throw std::invalid_argument("-i <INPUT> can't be empty");
+    } if (FLAGS_m.empty()) {
+        throw std::invalid_argument("-m <MODEL FILE> can't be empty");
```

Comment:
```suggestion
        throw std::invalid_argument{"-i <INPUT> can't be empty"};
    } if (FLAGS_m.empty()) {
        throw std::invalid_argument{"-m <MODEL FILE> can't be empty"};
```
To align with interactive_face_detection_demo